### PR TITLE
**Fix:** Fix datatable actions placements

### PR DESCRIPTION
**Pull request by fabien0102**
# Summary

Before:
image

image

After:
image

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/DataTableHeader/DataTableHeader.styled.ts [hunk 1]
```diff
@@ -40,13 +40,4 @@ export const ActionsContainer = styled.div`
   margin: auto;
   width: 100%;
   height: 100%;
-
-  /* The ContextMenu */
```
Comment (fabien0102) [inline src/DataTableHeader/DataTableHeader.styled.ts]: @TejasQ I'm not really sure why we introduce this peace of code, so let's check the preview with care 😅

File: src/useSticky/useSticky.ts [hunk 2]
```diff
@@ -40,7 +40,22 @@ const useSticky = ({
 
     if (node) {
       const rect = node.getBoundingClientRect()
-      setDisplaySettings({ position: "fixed", left: `${rect.left}px`, width: `${rect.width}px`, top: `${rect.top}px` })
+      const needOffset = rect.left + rect.width > window.innerWidth
+      if (needOffset) {
+        setDisplaySettings({
+          position: "fixed",
+          left: `${window.innerWidth - rect.width - 8}px`,
```
Comment (mpotomin) [inline src/useSticky/useSticky.ts]: Can we use https://github.com/contiamo/operational-ui/blob/af46bdcb42d72577bfcd5e1c925333c0ba4af3fc/src/utils/constants.ts#L198 instead?

Comment (fabien0102) [reply src/useSticky/useSticky.ts]: Sure

**Comment by mpotomin** [comment]
@fabien0102 Would it make sense to have a clickable test case to avoid regression in the future?

**Comment by mpotomin** [comment]
> @fabien0102 Would it make sense to have a clickable test case to avoid regression in the future?

found one in the DataTable!

**Pull request merged**